src/System.Threading.Tasks/src/System.Threading.Tasks.csproj

```diff
@@ -6,22 +6,31 @@
     <AssemblyName>System.Threading.Tasks</AssemblyName>
     <AssemblyVersion>4.0.11.0</AssemblyVersion>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
+    <!-- Monitor type-forward seen as duplicate type -->
```

Comment:
Not sure I understand that. The type-forward gets injected by genfacades correct? 

If it is coming from code we should try to scope the warning if possible with pragma's instead.